infra/gcp/clusters/projects/k8s-infra-public-pii/main.tf

```diff
@@ -0,0 +1,117 @@
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  project_id  = "k8s-infra-public-pii"
+  bucket_name = "k8s-infra-artifacts-gcslogs"
+}
+
+
+data "google_billing_account" "account" {
+  billing_account = "018801-93540E-22A20E"
+}
+
+data "google_organization" "org" {
+  domain = "kubernetes.io"
+}
+
+
+resource "google_project" "project" {
+  name            = local.project_id
+  project_id      = local.project_id
+  org_id          = data.google_organization.org.org_id
+  billing_account = data.google_billing_account.account.id
+}
+
+resource "google_project_service" "project" {
+  project = google_project.project.id
+
+  for_each = toset([
+    "bigquery.googleapis.com",
+    "logging.googleapis.com",
+    "storage-component.googleapis.com"
+  ])
+
+  service = each.key
+}
+
+
+// BigQuery dataset for audit logs
+resource "google_bigquery_dataset" "audit-logs-gcs" {
+  dataset_id                  = local.bucket_name
+  friendly_name               = local.bucket_name
+  delete_contents_on_destroy  = true
+  default_table_expiration_ms = 34560000000 #30 days
+  location                    = "US"
+  project                     = google_project.project.project_id
+}
+
+
+// Grant write access to the organizational sink
+resource "google_bigquery_dataset_iam_member" "bq-sinks-binding" {
+  project    = google_project.project.project_id
+  dataset_id = google_bigquery_dataset.audit-logs-gcs.id
+  role       = "roles/bigquery.dataEditor"
+  member     = google_logging_organization_sink.gcs-logs-org-sink.writer_identity
+
+}
+
+# A bucket to store logs in audit logs for GCS
+resource "google_storage_bucket" "audit-logs-gcs" {
+  name          = local.bucket_name
+  project       = google_project.project.project_id
+  storage_class = "REGIONAL"
+
+
+  lifecycle_rule {
+    action {
+      type = "Delete"
+    }
+
+    condition {
+      age = 400
+    }
+  }
+}
+
+// Grant write access to group-cloud-storage-analytics@google.com
+resource "google_storage_bucket_acl" "analytics_write" {
+  bucket      = google_storage_bucket.audit-logs-gcs.name
+  role_entity = ["WRITER:group-cloud-storage-analytics@google.com"]
+}
+
+
+// Allow ready-only access to k8s-infra-gcs-access-logs@kubernetes.io
+resource "google_storage_bucket_iam_member" "artificats-gcs-logs" {
+  bucket = google_storage_bucket.audit-logs-gcs.name
+  role   = "roles/storage.objectViewer"
+  member = "group:k8s-infra-gcs-access-logs@kubernetes.io"
+}
+
+// Create a sink for the organization
+resource "google_logging_organization_sink" "gcs-logs-org-sink" {
```

Comment:
Not familiar with this so no opinion

Reply:
I was looking at [Aggregated sinks](https://cloud.google.com/logging/docs/export/aggregated_sinks) but I realized I don't need this.